api/authorization.yml

```
@@ -116,7 +126,7 @@ components:
         - message
       properties:
         message:
-          description: short message explaining the error
+          description: Short message explaining the error
```

not related to your change - I don't think it have to be short
```suggestion
          description: The error message
```

---

changed